src/js/create_wordles.js

```diff
@@ -0,0 +1,118 @@
+"use strict";
```

Comment:
File name could probably more accurately be create_wordle. Also the appendNewWordleRow function could be be helpfully factored to be usable by the listWordles function. Note that in general we will want to reorganise this logic so that it is more stateless - if the API were designed differently, we could ensure that things did not jump around or get lost if the user refreshes the page.